gitpod-io/gitpod
# [observability] Team Workspace / Success Criteria dashboard -> no data

## Description

Success critiera dashboard for Team Workspace shows `no data` on page load

## Related Issue(s)

Fixes #7335

## How to test

To recreate the issue:
[Browse here](https://grafana.gitpod.io/d/jgjwvIc7k/success-criteria?orgId=1)

To recreate the fix:
1. Open a Gitpod workspace from this PR's branch
2. `kubectl port-forward svc/grafana 3000:3000`
3. [Start a workspace from here](https://kyleb-ws-dashboard.staging.gitpod-dev.com/workspaces)
4. Browse to Grafana and view the results

Or see the fix in action here:
<a href="https://www.loom.com/share/ece7afd53bc44302ae07f486931b14d5">
    <p>Fix for Team Workspace / Success Criteria dashboard -> no data #7335 - Watch Video</p>
    <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/ece7afd53bc44302ae07f486931b14d5-with-play.gif">
  </a>

## Release Notes

```release-note
Fix Team Workspace Success Criteria dashboard
```
